server/cli/start.go

```diff
@@ -41,21 +40,16 @@ var Context = server.NewContext()
 
 // An initCmd command initializes a new Cockroach cluster.
 var initCmd = &commander.Command{
-	UsageLine: "init <storage-location>",
+	UsageLine: "init -stores=(ssd=<data-dir>,hdd:7200rpm=<data-dir>|mem=<capacity-in-bytes>)[,...]",
```

Comment:
Are the attributes persisted during bootstrap? If so, we shouldn't allow passing them for `start` (unless we have a good plan of what the desired behaviour is when they change)